### ISSUE TITLE
Set signing profiles in iOs Project

### ISSUE THREAD

**Issue by aktivdigital-frontend** [ionic-team/trapeze#135]
Hi!

In iOs project, Can I set signing details/profiles with trapeze?

I use Stripe (from capacitor community) and every time when I run capacitor copy and sync, Xcode forget my signing details/profiles settings and I need do it again every time. In the case of Stripe, It need do it 12 times.

<img width="1001" alt="image" src="https://user-images.githubusercontent.com/103440937/205961874-746a7d16-a507-4e10-94aa-68bec95936c5.png">

Thank you for help!

**Comment by mlynch** [comment]
Not yet but it's something I could consider. Seems odd you'd have to do that after every sync though 🤔

**Comment by robingenz** [comment]
Yes — use the `ios.buildSettings` operation, which writes into the pbxproj build settings for the target and build you name (`CODE_SIGN_STYLE`, `DEVELOPMENT_TEAM`, `CODE_SIGN_IDENTITY`, `PROVISIONING_PROFILE_SPECIFIER`). That covers the `App` target. The extra targets you're seeing come from CocoaPods and live in `Pods.xcodeproj`, which trapeze doesn't touch; the usual fix there is a Podfile `post_install` hook, and Podfile support is tracked in #130. Closing this as answered — there's a config example in #209.
